diesel_infer_schema/src/inference.rs

```diff
@@ -146,20 +146,42 @@ pub fn load_foreign_key_constraints(database_url: &str, schema_name: Option<&str
     }
 }
 
+macro_rules! doc_comment {
```

Comment:
I think this is fine but @sgrif already emitted concerns about using macros for so few calls (2 in this case)